src/idfxt/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+
+set(SRC 
+  JSONDataInterface.cpp
+  JSONDataInterface.h
+  idd-full.h
```

Comment:
this holds an array of objects (by object type name) that are arrays of
field names, in order.  As noted earlier in this pull thread, generation
and on-going schema alteration will take further discussion

On Fri, Jan 16, 2015 at 6:40 PM, Dan Macumber <notifications@github.com>
wrote:

> In src/idfxt/CMakeLists.txt
> <https://github.com/NREL/EnergyPlus/pull/4636#discussion-diff-23117588>:
>
> > @@ -0,0 +1,14 @@
> > +
> > +set(SRC
> > +  JSONDataInterface.cpp
> > +  JSONDataInterface.h
> > +  idd-full.h
>
> @refaqtor <https://github.com/refaqtor> where is the code that generates
> the idd-full.h and idd-minimal-ordered.h files? What is the content of the
> idd-minimal-ordered.h file? Is that just object names or something?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NREL/EnergyPlus/pull/4636/files#r23117588>.
>